python/lsst/obs/subaru/isr.py

```diff
@@ -247,6 +247,11 @@ def runDataRef(self, sensorRef):
                 self.saturationDetection(ccdExposure, amp)
             if self.config.doSuspect and not badAmp:
                 self.suspectDetection(ccdExposure, amp)
+            ampMask = afwImage.MaskU(ccdExposure.getMaskedImage().getMask(), amp.getRawDataBBox(),
+                                     afwImage.PARENT)
+            maskVal = ampMask.getPlaneBitMask([self.config.saturatedMaskName, self.config.suspectMaskName])
+            if numpy.all(ampMask.getArray() & maskVal > 0):
+                badAmp = True
             if self.config.doOverscan and not badAmp:
```

Comment:
And my example of a single unsaturated pixel was an extreme, but I was wondering if it would be better to put a threshold of some sort on calling it a `badAmp` (e.g. if  `> threshold%` of all pixels are saturated and/or suspect, throw in the towel and call it bad).  No need to address that here, but could be something to keep in mind if it ever pops up in practice.